content/en/docs/concepts/services-networking/network-policies.md

```diff
@@ -312,6 +323,78 @@ The value of the label is the namespace name.
 While NetworkPolicy cannot target a namespace by its name with some object field, you can use the
 standardized label to target a specific namespace.
 
+## Pod lifecycle
+
+When a new NetworkPolicy object is created, it may take some for a network plugin
+to handle the new object. If the pod that is affected by a NetworkPolicy
+is created before the network plugin has completed NetworkPolicy handling,
+that pod may be started unprotected, and isolation rules will be applied when 
+the NetworkPolicy handling is completed.
+
+Once the NetworkPolicy is handled by a network plugin,
```

Comment:
```suggestion
For a cluster with a conformant networking plugin and a conformant implementation of
NetworkPolicy:
```

Reply:
(IMO we don't need this note; the entire page only applies to plugins that implement NetworkPolicy, and the entire `/services-networking/` section only (fully) applies to clusters with conformant networking plugins.)

Reply:
yeah good point, @sftim what do you think?

Reply:
> the entire page only applies to plugins that implement NetworkPolicy

The conceptual explanation is relevant even if your cluster is using a network plugin that has never heard of NetworkPolicy.

---

We should document _somewhere_ what we expect from a conforming network plugin. Otherwise, it's not fair on plugin authors - they need to know what's expected and shouldn't have to become detectives in meeting that end.

The way we inform implementers might once day be through a separate reference page, but for now this concept page is the closest thing we have.